### PR TITLE
Fix WCA event names

### PR DESCRIPTION
**Pull request by cubizh**
Updated list of event names to match the new naming convention.

### CODE REVIEW

**Comment by fw42** [comment]
Thanks for the fix! Do you have an official WCA list or PR to link to so that I can check that everything's correct?

**Comment by cubizh** [comment]
Here it is: https://www.worldcubeassociation.org/regulations/#9b

File: app/lib/wca_events.rb [hunk 1]
```diff
@@ -90,19 +90,19 @@
...
   {
-    name: "5x5 blindfolded",
+    name: "5x5 Blindfolded",
     handle: "5b",
     wca_handle: "555bf",
     formats: [ "Best of x" ]
   },
   {
-    name: "3x3 multi blind",
+    name: "3x3 Multi-Blind",
```
Comment (fw42) [inline app/lib/wca_events.rb]: Looks like this should be "3x3x3 Multi-Blind"? Same with 4x4 (4x4x4) and 5x5 (5x5x5)

**Comment by fw42** [comment]
I'm gonna fix the blindfolded mistake myself

**Comment by fw42** [comment]
Thanks for this PR!

**Pull request merged**